uwrt_mars_rover_hw/drivers/roboteq_driver/src/example_main.cpp

```diff
@@ -0,0 +1,164 @@
+
+  // using namespace std::chrono_literals;
+  // static constexpr auto EXECUTION_PERIOD{1s / 100};  // 100 Hz
+  // auto start_time = std::chrono::steady_clock::now();
+  // auto end_time = start_time + 5s;
+  // auto next_execution_time = start_time;
+  // while (std::chrono::steady_clock::now() < end_time) {
+  //   // read to all joints
+  //   for (int joint_num = 0; joint_num < NUM_DRIVE_JOINTS; joint_num++) {
+  //     positions[joint_num] = motor_controller.readAbsoluteEncoderCount(joint_num);
+  //     velocities[joint_num] = motor_controller.readEncoderMotorSpeed(joint_num);
+  //     efforts[joint_num] = motor_controller.readMotorAmps(joint_num);
+
+  //     static constexpr int NUMBER_FIELD_WIDTH{5};
+  //     std::cout << std::left << "joint_num: " << std::setw(NUMBER_FIELD_WIDTH) << joint_num
+  //               << "position: " << std::setw(NUMBER_FIELD_WIDTH) << positions[joint_num]
+  //               << "velocity: " << std::setw(NUMBER_FIELD_WIDTH) << velocities[joint_num]
+  //               << "effort: " << std::setw(NUMBER_FIELD_WIDTH) << efforts[joint_num] << std::endl;
+  //   }
+
+  //   // write to all joints
+  //   for (int joint_num = 0; joint_num < NUM_DRIVE_JOINTS; joint_num++) {
+  //     motor_controller.setMotorCommand(TEST_SPEED, joint_num);
+  //     bool command_successful = motor_controller.setVelocity(TEST_SPEED, joint_num);
+  //     if (!command_successful) {
+  //       bool shutdown_successful{false};
+  //       do {
+  //         shutdown_successful = motor_controller.emergencyShutdown();
+  //         std::cout << "Command write failed. Sending ESTOP!" << std::endl;
+  //       } while (!shutdown_successful);
+  //       std::cout << "Test FAILED. Aborting program early." << std::endl;
+  //       return EXIT_FAILURE;
+  //     }
+  //   }
+
+  //   next_execution_time += EXECUTION_PERIOD;
+  //   std::this_thread::sleep_until(next_execution_time);
+  // }
+
+  // // write to all joints
+  // for (int joint_num = 1; joint_num <= NUM_DRIVE_JOINTS; joint_num++) {
+  //   std::cout << "set motor command 0 on joint#" << joint_num << std::endl;
+  //   bool command_successful = motor_controller.setVelocity(0, joint_num);
+  //   if (!command_successful) {
+  //     bool shutdown_successful{false};
+  //     //      do {
+  //     std::cout << "Command write failed. Sending ESTOP!" << std::endl;
+  //     shutdown_successful = motor_controller.emergencyShutdown();
+  //     //      } while (!shutdown_successful);
+  //     std::cout << "Test FAILED. Aborting program early." << std::endl;
+  //     return EXIT_FAILURE;
+  //   }
+  // }
+
+  // motor_controller.stopInAllModes(1);
+  // motor_controller.stopInAllModes(2);
+
+  // for (int joint_num = 0; joint_num < NUM_DRIVE_JOINTS; joint_num++) {
+  //   bool success = motor_controller.stopInAllModes(joint_num);
+  //   assert(success);
+  //   success = motor_controller.stopInAllModes(joint_num);
+  //   assert(success);
+  //   (void)success;
+  // }
```

Comment:
Purpose of commented code?

Reply:
This was script to run outside of ROS for local testing of the driver library. I've deleted this file